pkg/sdk/config_test.go

```diff
@@ -60,33 +62,72 @@ func TestProfileConfig(t *testing.T) {
 	})
 
 	t.Run("with not found profile", func(t *testing.T) {
-		cleanupEnvVars := setupEnvVars(t, "", "", "", "", configPath)
-		t.Cleanup(cleanupEnvVars)
+		t.Setenv(snowflakeenvs.ConfigPath, configPath)
+
 		config, err := ProfileConfig("orgadmin")
 		require.NoError(t, err)
 		require.Nil(t, config)
 	})
+
+	t.Run("with not found config", func(t *testing.T) {
+		dir, err := os.UserHomeDir()
+		require.NoError(t, err)
+		t.Setenv(snowflakeenvs.ConfigPath, dir)
+
+		config, err := ProfileConfig("orgadmin")
+		require.Error(t, err)
+		require.Nil(t, config)
+	})
 }
 
-func TestEnvConfig(t *testing.T) {
-	t.Run("with no environment variables", func(t *testing.T) {
-		cleanupEnvVars := setupEnvVars(t, "", "", "", "", "")
-		t.Cleanup(cleanupEnvVars)
-		config := EnvConfig()
-		assert.Equal(t, "", config.Account)
-		assert.Equal(t, "", config.User)
-		assert.Equal(t, "", config.Password)
-		assert.Equal(t, "", config.Role)
+func Test_MergeConfig(t *testing.T) {
+	createConfig := func(user string, password string, account string, region string) *gosnowflake.Config {
+		return &gosnowflake.Config{
+			User:     user,
+			Password: password,
+			Account:  account,
+			Region:   region,
+		}
+	}
+
+	t.Run("merge configs", func(t *testing.T) {
+		config1 := createConfig("user", "password", "account", "")
+		config2 := createConfig("user2", "", "", "region2")
+
+		config := MergeConfig(config1, config2)
+
+		require.Equal(t, "user", config.User)
+		require.Equal(t, "password", config.Password)
+		require.Equal(t, "account", config.Account)
+		require.Equal(t, "region2", config.Region)
+		require.Equal(t, "", config.Role)
+
+		require.Equal(t, config1, config)
+		require.Equal(t, "user", config1.User)
+		require.Equal(t, "password", config1.Password)
+		require.Equal(t, "account", config1.Account)
+		require.Equal(t, "region2", config1.Region)
+		require.Equal(t, "", config1.Role)
 	})
 
-	t.Run("with environment variables", func(t *testing.T) {
-		cleanupEnvVars := setupEnvVars(t, "TEST_ACCOUNT", "TEST_USER", "abcd1234", "ACCOUNTADMIN", "")
-		t.Cleanup(cleanupEnvVars)
-		config := EnvConfig()
-		assert.Equal(t, "TEST_ACCOUNT", config.Account)
-		assert.Equal(t, "TEST_USER", config.User)
-		assert.Equal(t, "abcd1234", config.Password)
-		assert.Equal(t, "ACCOUNTADMIN", config.Role)
+	t.Run("merge configs reverted", func(t *testing.T) {
```

Comment:
i think the word is "inverted" not "reverted"

Reply:
Done.